control-plane/api/v1alpha1/gatewaypolicy_webhook_test.go

```diff
@@ -248,9 +250,14 @@ func TestGatewayPolicyWebhook_Handle(t *testing.T) {
 				},
 			})
 
-			require.Equal(t, tt.expAllow, response.Allowed)
+			assert.Equal(t, tt.expAllow, response.Allowed)
 			if tt.expErrMessage != "" {
-				require.Equal(t, tt.expErrMessage, string(response.AdmissionResponse.Result.Reason))
```

Comment:
`tt.expErrMessage` previously appeared in the `Reason` field but now appears in the `Message` field with the update to `controller-runtime`